tests/Catch2_Tests/CombinationTests.cpp

```diff
@@ -67,3 +67,64 @@ TEST_CASE("YouCanVerifyCombinationsOf9") {
                                                                const std::string& s9)
                               {return s1 + s2 + s3 + s4 + s5 + s6 + s7 + s8 + s9;}, letters, letters, letters, letters, letters, letters, letters, letters, letters);
 }
+
+TEST_CASE("YouCanVerifyCombinationsOf9Reports")
+{
+    auto d = clearFrontLoadedReporter();
+    FakeReporter reporter;
+    try
+    {
+        std::vector<std::string> letters{"a", "b"};
+        CombinationApprovals::verifyAllCombinations(
+            [](const std::string& s1, const std::string& s2, const std::string& s3, const std::string& s4, const std::string& s5,
+               const std::string& s6, const std::string& s7, const std::string& s8, const std::string& s9)
+            {return s1 + s2 + s3 + s4 + s5 + s6 + s7 + s8 + s9;},
+            reporter,
+            letters, letters, letters, letters, letters, letters, letters, letters, letters
+        );
+    }
+    catch(const ApprovalException&)
+    {
+        // ignore
+    }
+    REQUIRE(reporter.called == true);
+}
+
+TEST_CASE("YouCanVerifyCombinationsOf9ReportsOldParameterOrder")
```

Comment:
There's something I'm missing here...
There's a new test with 'OldParameterOrder' in the name, but the existing tests have not been changed - which suggests to me that they are also using the old order still.

Reply:
Yes, you're right, thanks!

However, there's actually just one test case that uses a custom reporter. For the others there really no change.

I can also duplicate that test to check both the old and new forms.

But let's first discuss where the reporter parameter should actually be. Before or after the converter?